run_ci.sh

```diff
@@ -45,7 +45,9 @@ mount | grep root-
 # The CI host has a "/" filesystem that reset for each job, the only partition
 # that persist is /opt (and /boot), we can use this to store data between jobs
 FILECACHEDIR=/opt/data/filecache
-FILESTOCACHE="/opt/dev-scripts/ironic/html/images/$RHCOS_IMAGE_FILENAME_OPENSTACK /opt/dev-scripts/ironic/html/images/ironic-python-agent.initramfs /opt/dev-scripts/ironic/html/images/ironic-python-agent.kernel"
+# Get the last openstack image
+LAST_OPENSTACK_IMAGE=$(basename $(ls -r $FILECACHEDIR/*-openstack.qcow2 | head -n1))
```

Comment:
I think we will still get the same error if the cache is empty. Maybe make sure the image exists here before adding it to FILESTOCACHE?

Reply:
Oh, good point!